sd_mining_core/utils/model_utils.py

```diff
@@ -135,7 +133,7 @@ def load_default_model(config):
     
     if config.specified_model_id:
         if config.specified_model_id not in model_ids:
-            print(f"Specified model ID {config.specified_model_id} not found locally. Exiting...")
+            logging.error(f"Specified model ID {config.specified_model_id} not found locally. Exiting...")
```

Comment:
Since most `print`s are replaced by logging.error in the log file, it might get harder for us to debug community miner problems as they may lose / override the log files. How about changing the logging warn/error functions to print logs to both console and files?